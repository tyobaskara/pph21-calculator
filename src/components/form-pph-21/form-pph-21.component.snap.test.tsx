import React from "react";
import renderer from 'react-test-renderer';
import FormPPH21Page from './form-pph-21.component';

test('FormPPH21Page snaptest', () => {
  const component = renderer.create(
    <FormPPH21Page/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});