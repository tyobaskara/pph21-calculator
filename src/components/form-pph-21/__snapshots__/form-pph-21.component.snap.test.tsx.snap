// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormPPH21Page snaptest 1`] = `
<div>
  <form
    className="form-pph21"
    onSubmit={[Function]}
  >
    <h1>
      PPH-21 Calculator
    </h1>
    <div
      className="form-group"
    >
      <label
        htmlFor="annualIncome"
      >
        Annual Income
      </label>
      <input
        id="annualIncome"
        name="annualIncome"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <label
        htmlFor="taxRelief"
      >
        Person Profile:
      </label>
      <select
        dir="rtl"
        id="taxRelief"
        name="taxRelief"
        onChange={[Function]}
        value="TK0"
      >
        <option
          value="TK0"
        >
          TK0 - Single
        </option>
        <option
          value="K0"
        >
          K0 - Married with no dependant
        </option>
        <option
          value="K1"
        >
          K1 - Married with 1 dependant
        </option>
        <option
          value="K2"
        >
          K2 - Married with 2 dependants
        </option>
        <option
          value="K3"
        >
          K3 - Married with 3 dependants
        </option>
      </select>
    </div>
    <div>
      <button
        type="submit"
      >
        Calculate
      </button>
    </div>
  </form>
  <div
    className="form-pph21-result"
  />
</div>
`;
